readme.md

```diff
@@ -93,6 +93,15 @@ $ npm test 		    	# Run tests
 $ npm test -- -u 		# Update Jest Snapshots
 ```
 
+#### Error Logs :
+
+Server Error logs are generated at /logs/error.log
+
+Run the command to read the logs using Bash (Linux) :
+
+`$ tail -f ./logs/error.log`
```

Comment:
we should simply log to standard output, also, we don't want to log only errors but also info messages and warnings.

Reply:
I have added error, warnings and info logs too